.eslintrc.js

```diff
@@ -196,6 +196,7 @@ module.exports = {
       rules: {
         "import/no-named-as-default-member": ["off"],
         "@intlify/vue-i18n/no-raw-text": ["off"],
+        "no-disabled-test": "off",
```

Comment:
The disabled rules need to be prefixed with the plugin name. `jest` in this case, `playwright` in the other. I've suggested adding a comment to help explain why the rule is disabled.

```suggestion
        // Superceeded by `@openverse/no-disabled-test`
        "jest/no-disabled-test": "off",
```